SH4RE/SH4RE/CreateListing/CreateListingView.swift

```diff
@@ -226,6 +224,20 @@ struct CreateListingView: View {
             }
             else {
                 calAvail.append(availabilitySelection)
+                let calendar = Calendar.current
+                let components = calendar.dateComponents([.year, .month, .day], from: Date())
+                let startOfMonth = calendar.date(from:components)!
+                let numberOfDays = calendar.range(of: .day, in: .quarter, for: startOfMonth)!.upperBound
+                let allDays = Array(0..<numberOfDays).map{ calendar.date(byAdding:.day, value: $0, to: startOfMonth)!}
+                if (availabilitySelection == "Everyday") {
+                    calAvail = []
+                }
+                else if (availabilitySelection == "Weekdays") {
+                    calAvail = allDays.filter{ calendar.isDateInWeekend($0) }
+                }
+                else if (availabilitySelection == "Weekends") {
+                    calAvail = allDays.filter{ !calendar.isDateInWeekend($0) }
```

Comment:
same as above, why grab dates that are NOT in the weekend, if avail is weekends?

Reply:
see above